app/views/steps/_form.html.erb

```diff
@@ -59,11 +59,67 @@
 </div>
 
 <div class="form-group">
-  <div>
-    <p>
-      <%= step.broken_links %>
-    </p>
-  </div>
+  <% if step.broken_links && step.broken_links.count > 0 %>
+    <div class="broken-links">
+      <div class="panel-group panel-warning" id="accordion" role="tablist" aria-multiselectable="true">
+        <div class="panel-heading">
+          <%= render 'shared/steps/broken-links-summary', step: step %>
+        </div>
+        <% step.broken_links.each.with_index do |link, index| %>
+          <div class="panel panel-warning">
+            <div class="panel-heading" role="tab" id="brokenLinkHeading<%= index %>">
+              <h4 class="panel-title">
+                <a role="button" data-toggle="collapse" data-parent="#accordion" href="#brokenLink<%= index %>" aria-expanded="true" aria-controls="brokenLink<%= index %>">
+                  <%= link['uri'] %>
+                </a>
+              </h4>
+            </div>
+            <div id="brokenLink<%= index %>" class="panel-collapse collapse" role="tabpanel" aria-labelledby="brokenLinkHeading<%= index %>">
+              <div class="panel-body">
+                <dl class="dl-horizontal">
+                  <dt>URI</dt>
+                  <dd><%= link['uri'] %></dd>
+                  
+                  <dt>Status</dt>
+                  <dd><%= link['status'] %></dd>
+                  
+                  <dt>Last checked</dt>
+                  <dd><%= link['checked'].to_datetime.to_formatted_s(:rfc822) %></dd>
+                  
+                  <dt>Problem summary</dt>
+                  <dd><%= link['problem_summary'] %></dd>
+                  
+                  <% if link['errors'] && link['errors'].count > 0 %>
+                    <dt>Errors</dt>
+                    <dd>
+                      <ul class="broken-links--list">
+                        <% link['errors'].each do |error| %>
+                          <li><%= error %></li>
+                        <% end %>
+                      </ul>
+                    </dd>
+                  <% end %>
+                  
+                  <% if link['warnings'] && link['warnings'].count > 0 %>
+                    <dt>Warnings</dt>
+                    <dd>
+                      <ul class="broken-links--list">
+                        <% link['warnings'].each do |warning| %>
+                          <li><%= warning %></li>
+                        <% end %>
+                      </ul>
+                    </dd>
+                  <% end %>
+                  
+                  <dt>Suggested fix</dt>
+                  <dd><%= link['suggested_fix'] %></dd>
```

Comment:
I'm curious to know what all the possible values for `suggested_fix` are (@jonodrew?).